packages/salesforcedx-vscode-soql/test/vscode-integration/testUtilities.ts

```diff
@@ -24,10 +24,21 @@ import {
   QueryDataViewService
 } from '../../src/queryDataView/queryDataViewService';
 
+const sfdxCoreExtension = vscode.extensions.getExtension(
```

Comment:
Same as above, let's stop using this pattern for depending on the core extension.